package.json

```diff
@@ -3,21 +3,30 @@
   "version": "0.0.1",
   "description": "Nutonian General Graphing Extension Tool",
   "author": "Nutonian, Inc.",
-  "repository": "https://bitbucket.org/nutonian/nugget",
+  "repository": "https://github.com/nutonian/nugget",
+  "scripts": {
+    "build": "webpack",
+    "build:dist": "NODE_ENV=production webpack",
+    "start": "concurrently --kill-others \"webpack --watch\" \"grunt development\"",
+    "test": "open http://localhost:9000/_SpecRunner.html"
+  },
   "devDependencies": {
```

Comment:
I updated packages as far as they could go, but `grunt-template-jasmine-requirejs` isn't maintained any more and doesn't work with Grunt 1.0.0